### PR TITLE
Done - home page

### PR DESCRIPTION
**Pull request by MuskanChhatrasal**

### CODE REVIEW

**Review by aayushjain-code — APPROVED** [review]
Try to make view all button centered else looks fine to me.!

File: src/Components/Footer/footer.js [hunk 1]
```diff
@@ -3,7 +3,25 @@ import './footer.css'
...
+          </a>
+          <a href="https://www.linkedin.com/in/muskan-chhatrasal/">
+            <i className="fab fa-linkedin"></i> linkedin
+          </a>
+        </div>
+      </div>
+
+      <div className="credit">by <span>Muskan</span> | all rights reserved</div>
+    </section>
+    </div>
```
Comment (partha8) [inline src/Components/Footer/footer.js:24]: suggestion: you can be more semantic here, there are a lot of divs :sweat_smile:

File: src/Components/Home/home.js [hunk 2]
```diff
@@ -1,9 +1,49 @@
...
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquam
+          libero nostrum veniam facere tempore nisi.
+        </p>
+        <a className="btn">shop now</a>
+      </div>
+    </section>
+
+
+
+
```
Comment (partha8) [inline src/Components/Home/home.js:21]: Nitpicking: There's a lot of space here

File: src/Components/Navbar/navbar.js [hunk 3]
```diff
@@ -1,9 +1,33 @@
...
+        <Link to='/'><i className='fas fa-shopping-basket'></i> Wow shop</Link>
+      </a>
+
+      <nav className="navbar">
+        <Link to='/'><a>home</a></Link>
+        {/* <a href="#speciality">Offer</a> */}
+        <Link to='/products'><a>products</a></Link>
+        <Link to='/wishlist'><a>Wishlist</a></Link>
+        <Link to='/cart'><a>Cart</a></Link>
+        {/* <a href="#home">Login</a> */}
```
Comment (partha8) [inline src/Components/Navbar/navbar.js:19]: If the commented out code is not needed in the future, you can remove it.

**Review by partha8 — APPROVED** [review]
Overall great work, good UI!

**Comment by MuskanChhatrasal** [comment]
Okayy!! Will do that

File: src/Components/Footer/footer.js (hunk 1 above)
Comment (MuskanChhatrasal) [reply src/Components/Footer/footer.js:24]: Okayy!! Will do that.

File: src/Components/Home/home.js (hunk 2 above)
Comment (MuskanChhatrasal) [reply src/Components/Home/home.js:21]: Will remove that. Thankyou!!

File: src/Components/Navbar/navbar.js (hunk 3 above)
Comment (MuskanChhatrasal) [reply src/Components/Navbar/navbar.js:19]: Okayy!!

**Pull request merged**